lua/pl/path.lua

```diff
@@ -292,21 +296,22 @@ function path.splitext(P)
     end
 end
 
---- return the directory part of a path
--- @string P A file path
+--- Get the directory part of a path.
+-- @string P A path
 -- @treturn string everything before the last dir-separator
 -- @see splitpath
 -- @usage
 -- path.dirname("/some/path/file.txt")   -- "/some/path"
 -- path.dirname("file.txt")              -- "" (empty string)
+-- path.dirname("/file.txt")             -- "" (empty string)
 function path.dirname(P)
     assert_string(1,P)
     local p1 = path.splitpath(P)
     return p1
 end
 
---- return the file part of a path
--- @string P A file path
+--- Get the base part of a path.
```

Comment:
This might actually be more confusing to some people. I suggest a wording that includes both labeling for clarify, perhaps:

```suggestion
--- Get the base part (filename segment) of a path.
```